tests/main/media-sharing/task.yaml

```diff
@@ -4,17 +4,16 @@ details: |
     the mount namespace used by snap applications into the main mount
     namespace.
 prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local_devmode test-snapd-tools
     mkdir -p /media/src
     mkdir -p /media/dst
     touch /media/src/canary
-    snap install --devmode snapd-hacker-toolbelt
 execute: |
-    export PATH=$PATH:/snap/bin
     test ! -e /media/dst/canary
-    snapd-hacker-toolbelt.busybox sh -c 'mount --bind /media/src /media/dst'
+    test-snapd-tools.cmd mount --bind /media/src /media/dst
```

Comment:
I think the one in the store has to be there as some tests install one from the store and some install the local one. The difference is that since you now have full assertions auto-connections take place. I think what is missing is a cron job that publishes the snaps from the tree into the store.

@fgimenez  ^^ what is your opinion?